### DISCUSSION TITLE
[bug] Version 5.0.4. Do not show the Switch Network modal when the app doesn't support your current network

### DISCUSSION THREAD

**Discussion by vietvh-3042** [reown-com/appkit#2509] [Q&A] [answered]
I noticed that from version 5.0.4. Modal Switch Network is not displayed when This app doesn't support your current network. I need this feature, is there any way to do it?
![image](https://github.com/WalletConnect/web3modal/assets/103104318/390847f5-91f3-4a04-874f-d51447b12707)

**Comment by glitch-txs** [comment]
mmmm could you try setting this to false?
https://docs.walletconnect.com/appkit/react/core/options#allowunsupportedchain

**Reply by vietvh-3042** [reply]
I tried it, but it doesn't seem to work. No modal appears when I switch to an unsupported network
![image](https://github.com/WalletConnect/web3modal/assets/103104318/b56305cb-e561-4e32-8f39-d4c8ffef8497)

**Reply by glitch-txs** [reply]
Could you open an issue?

**Reply by vietvh-3042** [reply, marked as answer]
I have opened the issue #2518

**Reply by Markcode1992** [reply]
Switch Network modal broken since 5.0.4 release and no effort to fix?
